debezium-sink/src/main/java/io/debezium/sink/batch/AbstractBuffer.java

```diff
@@ -6,16 +6,15 @@
 package io.debezium.sink.batch;
 
 import java.util.LinkedHashMap;
-import java.util.Map;
-import java.util.stream.Collectors;
+import java.util.List;
 
-import io.debezium.metadata.CollectionId;
 import io.debezium.sink.DebeziumSinkRecord;
 import io.debezium.sink.SinkConnectorConfig;
+import io.debezium.util.LinkedHashMapExtractor;
 
 public abstract class AbstractBuffer implements Buffer {
 
-    protected final Map<CollectionId, LinkedHashMap<Object, DebeziumSinkRecord>> records = new LinkedHashMap<>();
+    protected final LinkedHashMap<Object, DebeziumSinkRecord> records = new LinkedHashMap<>();
```

Comment:
Maybe it is possible to use `java.util.SequencedMap` in the decalration here and in other parts of the code?